pkg/minikube/download/preload.go

```diff
@@ -45,6 +45,15 @@ const (
 	PreloadVersion = "v11"
 	// PreloadBucket is the name of the GCS bucket where preloaded volume tarballs exist
 	PreloadBucket = "minikube-preloaded-volume-tarballs"
+
+	// Enumeration for preload existence cache.
```

Comment:
why not have boolean for all of these so it easier to read ?
either 
var exists *bool
or maybe a struct
```
type preloadStatus struct{
    exists *bool
}
```

then we could check 
```
if exists != nil && exists  {
...
```

Reply:
I'm not a big fan of the constants-as-enum strategy, but I also think pointer to bool does not work well, since in order to assign exists you have to do:
```
var t = <value>
exists = &t
```
(unless I'm missing something).
I also considered just several bools, one for unknown and one for exists. So `unknown = true` means the preload must be checked, `unknown = false; exists = false` means definitely missing, and `unknown = false; exists = true` means definitely present. The trouble here is when `unknown = true` exists can "float" which leaves room for bugs. Thoughts?

Reply:
thats fair we could do this way too